### DISCUSSION TITLE
画像表示のアニメーションについて

### DISCUSSION THREAD

**Discussion by xx-n59** [Futurealize-with-Tech/fwt#45] [General]
swiper.jsをインストールします！

どれが最適解かわからなくて数パターン作ってるのであとで選んで欲しいです、今日の夜までには載せます

**Comment by xx-n59** [comment]
https://github.com/Futurealize-with-Tech/fwt/assets/85819164/b20cfc6f-cd28-4a5c-8eea-35dc63d44742


https://github.com/Futurealize-with-Tech/fwt/assets/85819164/e8211e5b-2aba-4fc3-98ba-adfae8b92665

**Reply by xx-n59** [reply]
どちらが良いですか？時間ある時にお願いします

**Reply by am2525nyan** [reply]
二個目の方が好きです！

**Reply by Hasuro-R** [reply]
同じ2個目！

**Comment by Hasuro-R** [comment]
> swiper.jsをインストールします！

どうせframmer-motion入れるから、swiper.jsは入れる必要ないかも！（frammer-motionでも割と同じことできるので、バンドルサイズが増えるだけになる可能性が😥）

**Reply by xx-n59** [reply]
まじかーーー
じゃあframmer-motionの方でやり直します！
